### ISSUE TITLE
Add proverbs package to organization

### ISSUE THREAD

**Issue by andriisoldatenko** [go-proverbs/go-proverbs.github.io#34]
PoC https://github.com/andriisoldatenko/proverbs

```
go run github.com/andriisoldatenko/proverbs

Go Proverbs, by Rob Pike

Don't communicate by sharing memory, share memory by communicating.
Concurrency is not parallelism.
Channels orchestrate; mutexes serialize.
The bigger the interface, the weaker the abstraction.
Make the zero value useful.
interface{} says nothing.
Gofmt's style is no one's favorite, yet gofmt is everyone's favorite.
A little copying is better than a little dependency.
Syscall must always be guarded with build tags.
Cgo must always be guarded with build tags.
Cgo is not Go.
With the unsafe package there are no guarantees.
Clear is better than clever.
Reflection is never clear.
Errors are values.
Don't just check errors, handle them gracefully.
Design the architecture, name the components, document the details.
Documentation is for users.
Don't panic.
```

what do you think?

**Comment by andriisoldatenko** [comment]
@HarveyKandola what do you think?

**Comment by HarveyKandola** [comment]
Current website should be sufficient for reference purposes.

Not sure who will bother running package to get the proverbs list.

**Comment by andriisoldatenko** [comment]
inspired by https://www.python.org/dev/peps/pep-0020/
tl;dr
```
import this
```
or in golang
```
package main

import _ "github.com/go-proverbs/proverbs/pkg"

func main() {}
```

**Comment by andriisoldatenko** [comment]
asked in reddit, let's wait for feedback: https://www.reddit.com/r/golang/comments/d36ogv/what_do_you_think_to_add_something_similar_to/
